src/azure-cli/azure/cli/command_modules/vm/_params.py

```diff
@@ -789,6 +789,7 @@ def load_arguments(self, _):
                    help='Name of the extension.', id_part=None)
         c.argument('force_update', action='store_true', help='force to update even if the extension configuration has not changed.')
         c.argument('extension_instance_name', extension_instance_name_type)
+        c.argument('enable_automatic_upgrade', options_list=['--enable-automatic-upgrade', '-a'], action='store_true', help='Indicate the extension should be automatically upgraded by the platform if there is a newer version of the extension available.')
```

Comment:
Is there a way to disable it after it's enabled?

Reply:
`vm extension` does not support `update`. We can `set` it again.